added first letter capitalization validation to contact form

chelsey-g opened the pull request:
Fixes #7 -

I added a feature that will allow the user to only enter the first letter as capitalized in the name input. This will fix the problem that was occurring with contacts not being in alphabetical order. 

** I also removed the validation feature that does not allow the user to enter duplicate names.

phishy commented:
It's a little user-hostile to make an error for this. Maybe it's better just to capitalize it for them?

chelsey-g commented:
> It's a little user-hostile to make an error for this. Maybe it's better just to capitalize it for them?

Good idea!!!

chelsey-g commented:
I removed the function that prompts an error if the user does not enter a capitalized first letter in the name input. I rewrote it so the user does not have to see that error and it automatically changes to uppercase.
